fact/pixels.py

```diff
@@ -33,6 +33,14 @@
     )}
 
 
+def reorder_softid2chid(array):
+    '''
+    Returns view to the given array, remapped from softid order
+    to chid order (e.g. MARS ordering to fact-tools ordering)
+    '''
+    return array[CHID_2_SOFTID]
+
```

Comment:
Is this new function worth it?
with it, code might look like this:
```
import fact

bad = np.arange(1440)
good = fact.pixels.reorder_softid2chid(bad)
```

Without it code might look like this:
```
import fact

bad = np.arange(1440)
good = bad[fact.pixels.CHID_2_SOFTID]
```

I think both versions are about equally easy to understand when reading them.
Introducing the function, still allows people to use the version without the function, so one will find both variants possibly, this is a minor downside, when introducing a function, I think.